instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -374,6 +397,16 @@ def traced_execution(
             name, kind=SpanKind.CLIENT
         ) as span:
             self._populate_span(span, cursor, *args)
+            if args and self._commenter_enabled:
+                try:
+                    comment = self._generate_comment(span)
+                    if isinstance(args[0], bytes):
+                        comment = comment.encode("utf8")
+                    args_list = list(args)
+                    args_list[0] += comment
+                    args = tuple(args_list)
+                except Exception as exc:
+                    _logger.warning("Exception while generating sql comment: %s", exc)
```

Comment:
```suggestion
                    _logger.exception("Exception while generating sql comment: %s", exc)
```
Is [this](https://docs.python.org/3/library/logging.html#logging.Logger.exception) more appropriate?